### DISCUSSION TITLE
World Chunk Unloading

### DISCUSSION THREAD

**Discussion by kylegrahammatzen** [Minestom/Minestom#1708] [Help] [answered]
Does MineStom plan to incorporate any bulk chunk unloading event if multiple chunks were unloaded at once? or are they always fired in separate events.

**Comment by KrystilizeNevaDies** [comment, marked as answer]
TheMode and I have been working on an area API, and I have been working on abstracting instances more.

In the future I'd like to join these concepts however it will take a while and there is no guarantee that it will happen.

**Comment by kylegrahammatzen** [comment]
I see, appreciate the reply.
